### PR TITLE
Update Play Functionality

### PR DESCRIPTION
**Pull request by Kamsi-idimogu**
The play method in MXLPlayer.java is updated to play music from input guitar text tabs and drum text tabs.

### CODE REVIEW

**Comment by CCSCovenant** [comment]
It crashed on restTest.txt

File: src/main/java/player/MXLPlayer.java [hunk 1]
```diff
@@ -63,13 +66,124 @@ public String getPart(Part part,int measureID, int duration){
...
+			
+			if(durationCount < duration) {
+				durationCount += note.getDuration();
+			}
+			else {
+				if(note.getChord() == null && musicString.length() > 0 && musicString.charAt(musicString.length()-1) == '+') {
+					musicString.deleteCharAt(musicString.length()-1);
+					musicString.append(" ");
+				}
+				if(this.clef != null || measure.getAttributes().getClef() != null ) {
```
Comment (CCSCovenant) [inline src/main/java/player/MXLPlayer.java:82]: I don't see any code to handle with the rest note
rest should be represented as "R"

Also, I didn't see anything to handle the tied

**Review by CCSCovenant — CHANGES_REQUESTED** [review]
please see the comment

File: src/main/java/player/MXLPlayer.java (hunk 1 above)
Comment (Kamsi-idimogu) [reply src/main/java/player/MXLPlayer.java:82]: I've added support for rest notes and tied notes in my recent commit

**Review by CCSCovenant — APPROVED** [review]

**Pull request merged**